content/docs/command-reference/queue/remove.md

```diff
@@ -17,15 +17,14 @@ positional arguments:
 
 ## Description
 
-Removes the specified queued or completed experiment tasks from the queue. For
-completed tasks, this will also remove any associated output logs.
+Removes the specified queued or completed experiment `task`(s) from the queue.
+For completed tasks, this will also remove any associated output logs.
```

Comment:
Do we need to specify "queued or completed"? What other states are there? Only running, I suppose? If so, could phrase like "removes queued exp... Doesn't work with running experiments -- see `queue kill`".

Reply:
There will be other possible states after https://github.com/iterative/dvc/pull/8158

The previous definition of "running" was just "not queued and not completed", but after that PR, `RUNNING` state in DVC UI specifically means "executing pipeline". And there are other new states to represent setup and cleanup of the experiment.

So for the docs purposes, I think "queued or completed" is still the correct description here. It may also now be better to use something like "active" as the generic "not queued or completed" term instead of "running" (when it comes to things like "Doesn't work with running experiments")

Reply:
There are also failed experiments that probably should be added to the types of tasks that can be removed.

Active/inactive are new terms but seem intuitive. I'm fine with whatever you both decide.

Reply:
> So again, maybe better to phrase like "Removes non-active experiments from the queue. See `queue kill` to interrupt active ones" @dberenbaum ?

Sounds good, thanks!

Reply:
Updated in afd08c33f.

> explicitly document the states, maybe grouped by active vs. not

This is pending. Feel free to make an issue/task/note somewhere. Thanks